src/Message.cpp

```diff
@@ -583,19 +583,37 @@ void Message::rewind(bool complete)
 
 std::string Message::getInterfaceName() const
 {
-    return sd_bus_message_get_interface((sd_bus_message*)msg_);
+    auto interface = sd_bus_message_get_interface((sd_bus_message*)msg_);
```

Comment:
If we go for empty strings for nonexisting sd-bus message data, we could have this here (and in other methods in this PR):

```suggestion
    auto interface = sd_bus_message_get_interface((sd_bus_message*)msg_);
    return interface != nullptr ? {interface} : {};
```